recipes/vvenc/all/conanfile.py

```diff
@@ -0,0 +1,72 @@
+import os
+from conan.tools.build import stdcpp_library, check_min_cppstd  #, check_max_cppstd
+from conan import ConanFile, conan_version
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.files import get, copy, rmdir
+# from conan.errors import ConanInvalidConfiguration
+
+
+class vvencRecipe(ConanFile):
+    name = "vvenc"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Fraunhofer Versatile Video Encoder (VVenC)"
+    topics = ("video", "encoder", "codec", "vvc", "h266")
+    homepage = "https://www.hhi.fraunhofer.de/en/departments/vca/technologies-and-solutions/h266-vvc.html"
+    package_type = "library"
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    def validate(self):
+        if conan_version.major < 2:
+            pass # FIXME : uncomment later: raise ConanInvalidConfiguration("conan v1 is not supported")
```

Comment:
@SSE4 Let's drop this validate function to see where it'd fail for Conan 1.x. I can help you to fix those problems if you see them.

Reply:
it's dropped already (you see code is commented)

Reply:
I meant the whole `validate()` function.

Reply:
I really puzzled now, before [fd5face](https://github.com/conan-io/conan-center-index/pull/17025/commits/fd5faceb287121c70e1a349321f26b5ed1c17b3f) commit was green, but now it started to fail with no functional changes. I'll reject all the failed V1 configurations for now.